src/borg/archiver.py

```diff
@@ -976,6 +994,17 @@ def _info_repository(self, repository, key, cache):
         print(DASHES)
         print(STATS_HEADER)
         print(str(cache))
+
+        if export_file:
+            repository_info = {
+                'encripted': encrypted,
```

Comment:
`'encrypted'` (typo)

Probably better to make this more machine-readable, eg. like this:

```
'encryption': {
  'type': key.NAME,
  'keyfile': key.find_key(),
}
```

Unencrypted repo -> no 'encryption' dict?